### PR TITLE
Dx diagram new api descriptions

### PR DESCRIPTION
**Pull request by LanaDX**

### CODE REVIEW

File: api-reference/10 UI Widgets/dxDiagram/1 Configuration/historyToolbar/historyToolbar.md [hunk 1]
```diff
@@ -5,7 +5,11 @@ default: {}
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Configures the history toolbar settings.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Widgets/dxDiagram/1 Configuration/historyToolbar/historyToolbar.md]:
```suggestion
Configures the history toolbar's settings.
```

File: api-reference/10 UI Widgets/dxDiagram/4 Events/customCommandExecuted.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ type: eventType
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Raised after a custom command item has been clicked and allows you to perform custom actions.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Widgets/dxDiagram/4 Events/customCommandExecuted.md]:
```suggestion
Raised after a custom command item was clicked and allows you to perform custom actions.
```

Comment (dirk-pieterse) [reply api-reference/10 UI Widgets/dxDiagram/4 Events/customCommandExecuted.md]: "to perform custom actions" this is a bit vague

File: api-reference/10 UI Widgets/dxDiagram/1 Configuration/propertiesPanel/tabs/title.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ type: String
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies a title of the tab.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+The title and a tab header are not displayed when there is only one tab in the Properties panel.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Widgets/dxDiagram/1 Configuration/propertiesPanel/tabs/title.md]:
```suggestion
Titles and tab headers are not displayed when there is only one tab in the Properties panel.
```

File: api-reference/10 UI Widgets/dxDiagram/1 Configuration/propertiesPanel/tabs/title.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ type: String
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies a title of the tab.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Widgets/dxDiagram/1 Configuration/propertiesPanel/tabs/title.md]:
```suggestion
Specifies the tab's title.
```

File: api-reference/10 UI Widgets/dxDiagram/1 Configuration/propertiesPanel/tabs/commands.md [hunk 5]
```diff
@@ -5,7 +5,7 @@ type: Array<dxDiagramCustomCommand> | Array<String>
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Lists commands in a tab.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+You can populate a tab with commands or with groups of commands. When the [groups](/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/propertiesPanel/tabs/groups/) option is specified, the **commands** option is not in effect.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Widgets/dxDiagram/1 Configuration/propertiesPanel/tabs/commands.md]: What is the difference between 'commands or groups of commands'?

File: api-reference/10 UI Widgets/dxDiagram/1 Configuration/onCustomCommandExecuted.md [hunk 6]
```diff
@@ -5,19 +5,18 @@ EventForAction: dxDiagram.customCommandExecuted
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+A function that is executed after a custom command item has been clicked and allows you to perform custom actions.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Widgets/dxDiagram/1 Configuration/onCustomCommandExecuted.md]:
```suggestion
A function that is executed after a custom command item was clicked and allows you to perform custom actions.
```

Comment (dirk-pieterse) [reply api-reference/10 UI Widgets/dxDiagram/1 Configuration/onCustomCommandExecuted.md]: See the comment below

**Review by dirk-pieterse — APPROVED** [review]

File: api-reference/10 UI Widgets/dxDiagram/1 Configuration/propertiesPanel/tabs/commands.md (hunk 5 above)
Comment (LanaDX) [reply api-reference/10 UI Widgets/dxDiagram/1 Configuration/propertiesPanel/tabs/commands.md]: Perhaps:
Use the  [groups] option to group commands in the tab. Note, if the [groups] option is specified, the **commands** option is not in effect.

File: api-reference/10 UI Widgets/dxDiagram/4 Events/customCommandExecuted.md (hunk 2 above)
Comment (LanaDX) [reply api-reference/10 UI Widgets/dxDiagram/4 Events/customCommandExecuted.md]: perhaps: "to implement the custom command's logic."

File: api-reference/10 UI Widgets/dxDiagram/1 Configuration/propertiesPanel/tabs/commands.md (hunk 5 above)
Comment (dirk-pieterse) [reply api-reference/10 UI Widgets/dxDiagram/1 Configuration/propertiesPanel/tabs/commands.md]: Ok

File: api-reference/10 UI Widgets/dxDiagram/4 Events/customCommandExecuted.md (hunk 2 above)
Comment (dirk-pieterse) [reply api-reference/10 UI Widgets/dxDiagram/4 Events/customCommandExecuted.md]: Ok

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**